vr-wordle/src/Components/Grid.js

```diff
@@ -25,7 +25,7 @@ function StaticLetter(props) {
 
   const springs = useSpring({
     color: props.submitted ? color : "#3a3a3c",
-    delay: 700 + props.index * 500,
+    delay: 600 + props.index * 600,
     config: { duration: 200 },
   });
```

Comment:
I like this, how did you come up with these numbers?

Reply:
just trial and error!